hpcgap/lib/mapping.gi

```diff
@@ -195,9 +202,15 @@ InstallGlobalFunction( Image, function ( arg )
 
         return [];
 
+      else
+        ErrorNoReturn( "families of <elm> or <coll> and Source(<map>) ",
```

Comment:
Again, in this error message, the variable `<coll>` that it's referring to doesn't exist, so I think you should use words instead. Something like `...the families of the element or collection <elm> and Source(<map>)...`